packages/api-explorer/src/JsonForm/not-custom-editor.js

```diff
@@ -0,0 +1,18 @@
+/* eslint-disable no-underscore-dangle */
+const baseCustomEditor = require('./get-custom-editor')
+
+const isArrayWithItems = schema => schema.type === 'array' && schema.items
+const isObjectWithProps = schema => schema.type === 'object' && schema.properties
+
+module.exports = () => baseCustomEditor('multiple').extend({
+  preBuild() {
+    const { not: notSchema } = this.schema
+    if (isArrayWithItems(notSchema) || isObjectWithProps(notSchema)) {
```

Comment:
We should invert this condition to avoid code repetition

Reply:
done